service.onidle/addon.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="service.onidle" version="0.0.4" name="OnIdle" provider-name="Lunatixz">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+    </requires>
+    <extension point="xbmc.python.script" library="settings.py"/>
+    <extension point="xbmc.service" library="service.py" start="startup"/>
```

Comment:
@Lunatixz this is not a false positive: https://github.com/xbmc/addon-check/commit/33c3fa1618d3c42088212b4bd202115b4022fafd